mindsdb/libs/controllers/predictor.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, name, root_folder=CONFIG.MINDSDB_STORAGE_PATH, log_level=CONF
         if CONFIG.CHECK_FOR_UPDATES:
             try:
                 check_for_updates()
-            except:
+            except Exception:
```

Comment:
What's the point of explicitly catch `Exception` if we don't log it (e.g. `except Exception as e` + `print(e)`)  ?

I'm not saying we should log them, just not sure why making the `Exception` explicit is superior.

Reply:
This will swallow `KeyBoardInterrupt`:
```
try:
<whatever>
except:
<handle>
```
`except Exception` will not.

It's just annoying to have code not react to your Ctrl+C when running. Also just a good practice.

You can try this code:
```
while True:
  try:
   pass
  except:
   pass
```
This will hang forever and nothing can unhang it (except process kill). If you add `except Exception` it's fixed.

Thought it would be better to handle all exceptions of course.